components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/messaging-mechanism.adoc

```diff
@@ -50,7 +50,7 @@ The hash of an L2 → L1 message is computed on L1 as follows:
 keccak256(
     abi.encodePacked(
         FromAddress,
-        ToAddress,
+        uint256(ToAddress),
```

Comment:
@odednaor This is just pseudo code. Why do we need to state the type for this one specific parameter? (Also, there's a similar functions further on below on the same page.)